spec/registry.md

```diff
@@ -186,3 +186,17 @@ which only expects the `accord` option:
 
     let $obj = {$object :noun case=nominative}
     {You see {$color :adjective article=indefinite accord=$obj} {$obj}!}
+
+## Built-in Registry
+
+As hinted in the example above, implementations may <!-- FIXME: may vs should --> provide built-in functions to their users in order to perform common operations in a standardized manner instead of having to rely on custom functions for everything. The working group agrees that these builtins provided by implementations should expose the following capabilities to their users:
+
+* Ability to format the following types of data in a locale-sensitive manner:
+    * Dates and times
+    * Numbers
+    * Lists
+* Ability to select for the following types of information in a locale-sensitive manner:
+    * Plural rules
```

Comment:
The "basic" or default selector (whose name we are discussing in #433) is what ICU4J calls `SelectFormat`. That should be listed here.

Try to make a list of items that could be selectors. Maybe start by making a _long_ list of everything and we can try to triage the list by crossing things out (rather than trying to remember if we decided not to include it... 😉). My back of envelope for selectors would maybe look like:

* select/equals (string/value equality; can be used to build things like a poor person's gender selector)
* plural
* ordinal

Potentially below the line:
* grammatical gender
* case
* date/time query (before/after/has-field-value)
* choice (see Java's `ChoiceFormat`)